camayoc/tests/qpc/ui/conftest.py

```diff
@@ -94,5 +110,13 @@ def credentials(browser, qpc_login):
     options['credential_type'] = 'VCenter'
     create_credential(browser, options)
     yield names
-    browser.refresh()
-    delete_credential(browser, set(names.values()))
+    # Some tests are flaky, so we need to do a full clear
+    # to garbage-collect resources from failed calls.
+    # A clear-all option doesn't exit in the UI,
+    # So it is done through the CLI instead.
+    # Also, an internal server error sometimes occurs when wiping credentials
+    # https://github.com/quipucords/quipucords/issues/1275
+    try:
+        cleanup_server()
+    except Exception:
+        cleanup_server()
```

Comment:
:grimacing:

Reply:
yeah....

Reply:
List some of the expected exceptions, much better than capturing everything blindly.